examples/protocol_specification_ex/fipa.yaml

```diff
@@ -32,3 +32,16 @@ ct:Query: |
 ct:Description: |
   bytes description = 1;
 ...
+---
+reply:
+  cfp: [propose, decline]
+  propose: [accept, accept_w_inform, decline]
+  accept: [decline, match_accept, match_accept_w_inform, inform]
+  accept_w_inform: [decline, match_accept, match_accept_w_inform, inform]
```

Comment:
I think we should remove inform as a valid reply in both cases.

Reply:
Done